bin/templates/cordova/lib/prepare.js

```diff
@@ -214,7 +215,11 @@ function updateProjectAccordingTo (platformConfig, locations) {
     shell.sed(/package [\w\.]*;/, 'package ' + pkg + ';', java_files[0]).to(destFile);
     events.emit('verbose', 'Wrote out Android package name "' + pkg + '" to ' + destFile);
 
-    if (orig_pkg !== pkg) {
+    var removeOrigPkg = checkReqs.isWindows() || checkReqs.isDarwin() ?
+        orig_pkg.toUpperCase() !== pkg.toUpperCase() :
+        orig_pkg !== pkg;
```

Comment:
I know the naming of these variables didn't originate from you, but while we're here, can we also update the variable names used here? I found `orig_pkg` and `pkg` to be not very descriptive. Maybe `manifest_id` instead of `orig_pkg`?